app/views/components/docs/content-metadata.yml

```diff
@@ -0,0 +1,14 @@
+name: Content Metadata
+description: Lists publication dates and publishers.
+shared_accessibility_criteria:
+- link
+
+examples:
+  default:
+    data:
+      from:  ["<a href='/government/organisations/ministry-of-defence'>Ministry of Defence</a>"]
+      published: "31 July 2017"
+      last_updated: "20 September 2017"
+  multiple_publishers:
```

Comment:
Can we add another example with more than 2? Perhaps that would involve renaming this to `two_publishers`

Reply:
This may be personal preference, but I'd also include the published dates in these examples to see how all the elements of the component work together. You could always include a separate example, e.g `no_published_dates` to show an example on it's own.

Reply:
I think I need to double check the expected behaviour here - I thought that this component was not going to expand the dates and that was a possibility elsewhere on the page - however the screenshot on the trello card shows otherwise.

Reply:
I don't think this component need to expand the dates, but it should be able to contain a link to the #history anchor on the page (which will expand the dates).